features/provide_vat_receipt.feature

```diff
@@ -0,0 +1,33 @@
+@US259
+Feature: Add VAT receipt at the bottom of the printed summary page
```

Comment:
I think as a 'nice to have' in the future we will want to have the VAT receipt printable separately but as an MVP having the VAT receipt on the summary and on the printout is acceptable.